app/src/main/scala/org/alephium/explorer/api/EndpointExamples.scala

```diff
@@ -204,6 +205,29 @@ object EndpointExamples extends EndpointsExamples {
       value     = 60
     )
 
+  private val explainResult =
+    ExplainResult(
+      queryName  = "queryName",
+      queryInput = "Pagination(0,20,false)",
+      explain = Vector(
+        "Seq Scan on table_name  (cost=0.00..850.88 rows=20088 width=198) (actual time=0.007..4.358 rows=20088 loops=1)",
+        "  Filter: table_column",
+        "Planning Time: 0.694 ms",
+        "Execution Time: 5.432 ms"
+      ),
+      messages = Array(
+        "Used table_column_idx = false",
+        "Used table_pk         = true"
+      ),
+      passed = true
+    )
```

Comment:
it's weird it doesn't show int the  openapi.json.

Reply:
oh it's because it's not in `Documentation`, but still it's nice to have the example in the code.